### PR TITLE
feat(case management): added integration tests to NodeJS SDK

### PR DESCRIPTION
**Pull request by huydoan2**
## PR summary

**Fixes:** <! -- link to issue -->

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [ ] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [ ] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [ ] No

### CODE REVIEW

File: test/integration/case-management.v1.test.js [hunk 1]
```diff
@@ -0,0 +1,487 @@
...
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+'use strict';
+
+const CaseManagementV1 = require('../../dist/case-management/v1');
+const authHelper = require('../resources/auth-helper.js');
+
+// testcase timeout value (60s).
```
Comment (ycs334) [inline test/integration/case-management.v1.test.js:22]: Minor but the comment needs to be 120s or the timeout value should be 60000.

File: test/integration/case-management.v1.test.js [hunk 2]
```diff
@@ -0,0 +1,487 @@
...
+    },
+  ];
+
+  const attachmentPayload = {
+    filename: 'NodeJS SDK test file.png',
+    data: Buffer.from('This is a mock file.', 'utf8'),
+    contentType: 'image/png',
+  };
+
+  let attachmentId;
```
Comment (ycs334) [inline test/integration/case-management.v1.test.js:73]: I would improve legibility if constants and variables are bunched together into their own group. (e.g. all `const` variables at the top followed by `let` ones)

File: test/integration/case-management.v1.test.js [hunk 3]
```diff
@@ -0,0 +1,487 @@
...
+      key: 'cloud-object-storage',
+    },
+  };
+
+  const resourceCrn = 'crn:v1:staging:public:cloud-object-storage:global:a/19c52e57800c4d8bb9aefc66b3e49755:61848e72-6ba6-415e-84e2-91f3915e194d::';
+
+  const watchlistPayload = [
+    {
+      realm: 'IBMid',
+      user_id: 'ashwini.pc@ibm.com',
```
Comment (ycs334) [inline test/integration/case-management.v1.test.js:59]: I suggest we avoid test-account specific values for two reasons. Our tests will break if the test account gets modified. And our tests won't pass if another test account is used.

Comment (huydoan2) [reply test/integration/case-management.v1.test.js:59]: This is a hard issue. Business logic requires that only users in the account to be added to the watchlist.

Although account user modification happens, it doesn't happen frequently.  I've seen account specific details being used in newrelic alerts as well.

I would say that we delegate this account as a test account and no one should make any modification to its without careful consideration of all aspects.

File: test/integration/case-management.v1.test.js (hunk 2 above)
Comment (huydoan2) [reply test/integration/case-management.v1.test.js:73]: 👍

File: test/integration/case-management.v1.test.js (hunk 1 above)
Comment (huydoan2) [reply test/integration/case-management.v1.test.js:22]: 👍

**Pull request merged**